src/math/Box3.js

```diff
@@ -467,6 +467,9 @@ Object.assign( Box3.prototype, {
 	}(),
 
 	translate: function ( offset ) {
+		if ( ! offset.isVector3 ) {
+            throw new Error( 'THREE.Box3: .translate() expects a Vector3.' );
```

Comment:
Indentations not correct here...

Reply:
I hope to have cleaned all of them.

Reply:
BTW. You can use an ESLint plugin in your editor or IDE for better code style. `three.js` uses an ESLint preset that is configured in the package JSON. Linter plugins like the ESLint plugin for Atom automatically detect this present and show appropriate feedback in the editor.

Reply:
OK. ESLint plugin installed, this will definitely make things easier for me in the future.

I think code style in this PR is good already (ESList also does not show any issues to me). Or are there still some issues left?

Reply:
I thinks it's okay now.